packages/contextual-menu-react/src/ContextualMenuItem.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChevronRightIcon } from "@fremtind/jkl-icons-react";
+import cn from "classnames";
+import React, { ReactNode } from "react";
+interface ContextualMenuItemProps {
+    description: string;
+    expandable?: boolean;
+    divider?: boolean;
+    icon?: ReactNode;
+}
+
+export const ContextualMenuItem = ({
+    description,
+    icon,
+    expandable = false,
+    divider = false,
```

Comment:
Kunne man kanskje hatt en egen `Divider`-komponent, eventuelt en mulighet for å gruppere elementer à la [`<optgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup)? Skillelinjen hører jo strengt tatt ikke til menypunktet i seg selv, men er heller _mellom samlinger av_ elementer.

Dette er potensielt en litt større endring, så er ikke en blokker for min del. Samtidig vil jo en endring senere potensielt bety breaking change, så det kan i så fall være verdt å gjøre oppmerksom på i dokumentasjonen. Vet ikke om @wkillerud har noen innspill her?

Reply:
Enig i at `divider`-propen godt kan være sin egen komponent. Semantisk kan det være en `<hr />`. Se for eksempel [Bootstrap](https://getbootstrap.com/docs/5.3/components/dropdowns/#dividers).

```diff
<ContextualMenu triggerElement={<ContextualMenuTriggerIcon icon={<CheckIcon bold />} />}>
    <ContextualMenuItem description="Menyvalg 1" icon={<InfoIcon />} />
    <ContextualMenuItem description="Menyvalg 2" />
    <ContextualMenuItem 
        description="Menyvalg med en lengre beskrivende tekst" 
    />
+   <ContextualMenuDivider />
    <ContextualMenu
        openOnHover
        initialPlacement="right-start"
        triggerElement={
            <ContextualMenuItem 
                description="Ekspanderende menyvalg"
-               divider
                expandable  
            />
        }
    >
        <ContextualMenuItem description="Ekspandert menyvalg" icon={<CopyIcon />} />
        <ContextualMenuItem description="Ekspandert menyvalg med mer tekst" />
    </ContextualMenu>
    <ContextualMenu
        openOnHover
        initialPlacement="right-start"
        triggerElement={
            <ContextualMenuItem 
                icon={<InfoIcon />} 
                description="Ekspanderende menyvalg" 
                expandable 
            />
        }
    >
        <ContextualMenuItem description="Ekspandert menyvalg" icon={<CopyIcon />} />
        <ContextualMenuItem description="Ekspandert menyvalg med mer tekst" />
    </ContextualMenu>
</ContextualMenu>
```